### ISSUE TITLE
Fail to deploy to Local Git

### ISSUE THREAD

**Issue by v-mengwe** [microsoft/vscode-azurefunctions#2093]
**OS:** Mac/Windows
**Build Version:** [20200513.5](https://dev.azure.com/ms-azuretools/AzCode/_build/results?buildId=19233&view=results)

**Repro Steps:**

1. Open a project on VSCode-->Initialize the local Git repository by "git init"
2. Make sure your changes are committed in git
3. On an existing function app, change deployment source to local git
4. Deploy

**Expect:**
Deploy successfully.

**Actual:**
Fail to deploy. Please get the detailed log from \\trinlab2\BAQAFiles\CloudToolsTest\v-mengwe\localgit_logs.txt.

**Comment by EricJizbaMSFT** [comment]
Yeah I see the same error if I deploy a C# project using local git. This is the important bit I think:

> Command '...' was aborted due to no output nor CPU activity for 60 seconds. You can increase the SCM_COMMAND_IDLE_TIMEOUT app setting (or WEBJOBS_IDLE_TIMEOUT if this is a WebJob) if needed.

However, it worked if I made another commit and deployed again. I think it's populating a nuget cache for the first time (or something like that) and it's just taking longer than 60 seconds. As the log says - users can increase the timeout or try deploying again like I did and hopefully it should work. I tried with a JavaScript project and had no problems.

It's almost certainly an upstream bug, but it's pretty annoying that a super simple C# project hits this error on first deploy. On the plus side, local git is pretty unpopular on functions and actually [not recommended](https://docs.microsoft.com/en-us/azure/azure-functions/functions-deployment-technologies#local-git). Could be more important if we make remote build more prominent in https://github.com/microsoft/vscode-azurefunctions/issues/1847
